### PR TITLE
clarified tool vs sparktool

### PR DESCRIPTION
**Pull request by akiezun**
fixes #https://github.com/broadinstitute/gatk/issues/1286

@cmnbroad can you review?

### CODE REVIEW

File: gatk-launch [hunk 1]
```diff
@@ -34,18 +34,17 @@ def main(args):
         if len(args) is 0:
             print("")
             print("Usage:")
-            print(" General format")
-            print("    gatk-launch Tool toolArgs -- sparkArgs")
+            print(" Usage template all tools (runs direct runner when used with a Spark tool)")
```
Comment (cmnbroad) [inline gatk-launch]: Usage template *for* all tools

Comment (cmnbroad) [inline gatk-launch]: I would also put "direct" in all caps here for symmetry with the other uses.

Comment (cmnbroad) [inline gatk-launch]: My  2 cents, I would say:

"Usage template for all tools (uses --sparkRunner DIRECT when used with a Spark tool)"

Overall I think it looks much better.

Comment (cmnbroad) [inline gatk-launch]: Overall I think it looks much better. Back to @akiezun.

Comment (akiezun) [inline gatk-launch]: fixed

**Pull request merged**